test/support/builders/models/submission/CISSubmissionBuilder.scala

```diff
@@ -30,19 +30,15 @@ object CISSubmissionBuilder {
 
   val anUpdateCISSubmission: CISSubmission = CISSubmission(
     employerRef = None,
-    submissionId = Some("submissionId"),
     contractorName = None,
-    periodData = Seq(
-      aPeriodData
-    )
+    periodData = Seq(aPeriodData),
+    submissionId = Some("submissionId")
   )
 
   val aCreateCISSubmission: CISSubmission = CISSubmission(
```

Comment:
Please make this class a proper test builder. Additional methods you need should be done where you use the builder.

Reply:
updated